include/depthai/openvino/OpenVINO.hpp

```diff
@@ -54,6 +54,59 @@ class OpenVINO {
         std::vector<uint8_t> data;
     };
 
+    /**
+     * @brief A superblob is an efficient way of storing generated blobs for all different number of shaves.
+     */
+    class Superblob {
+       public:
+        /** Number of patches in a superblob*/
+        static constexpr size_t NUMBER_OF_PATCHES = 16;
+
+        /**
+         * @brief Construct a new Superblob object
+         *
+         * @param pathToSuperblobFile: Path to the superblob file (.superblob suffix)
+         */
+        Superblob(const std::string& pathToSuperblobFile);
+
+        /**
+         * @brief Generate a blob with a specific number of shaves
+         *
+         * @param nShaves: Number of shaves to generate the blob for. Must be between 1 and NUMBER_OF_PATCHES.
+         * @return dai::OpenVINO::Blob: Blob compiled for the specified number of shaves
+         */
+        dai::OpenVINO::Blob getBlobWithNShaves(int nShaves);
```

Comment:
```suggestion
        dai::OpenVINO::Blob getBlobWithNumShaves(int numShaves);
```

Nitpick, I think elsewhere `num` is used as a shorthand

Reply:
Note, function name included for the suggested change, not just func arg name @lnotspotl

Reply:
I have accidentally changed it back, my bad. Fixed in f260f3b9a8843af5135d8dfb11045c1185bbeb53. Thanks for pointing that out @themarpe